ocs_ci/deployment/vmware.py

```diff
@@ -598,12 +598,26 @@ def destroy_cluster(self, log_level="DEBUG"):
         """
         previous_dir = os.getcwd()
 
-        # Download terraform binary based on ocp version and
-        # update the installer path in ENV_DATA
-        # use "0.11.14" for releases below OCP 4.5
-        # TODO: For cluster installed by old version of terraform we need to
-        # still run old version for destroy ( upgrade scenario )
-        terraform_version = config.DEPLOYMENT['terraform_version']
+        # Download terraform binary based on terraform version
+        # in terraform.log
+        terraform_log_path = os.path.join(
+            config.ENV_DATA.get('cluster_path'),
+            config.ENV_DATA.get('TF_LOG_FILE')
+        )
+
+        # check for terraform.log, this check is for partially
+        # deployed clusters
+        try:
+            with open(terraform_log_path, 'r') as fd:
+                logger.debug(
+                    f"Reading terraform version from {terraform_log_path}"
+                )
+                version_line = fd.readline()
```

Comment:
Is the version guaranteed to be on the first line in the log file?

Reply:
Yes, version will be always fist line of the file